packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -0,0 +1,61 @@
+import { LightningElement, api } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+
+
+export default class QuanticTab extends LightningElement {
+  /** @type {import("coveo").Tab} */
+  tab;
+  /** @type {()=> void} */
+  unsubscribe;
+  /** @type {boolean} */
+  isActive;
+  /** @type {string} */
+  @api label;
+  /** @type {string} */
+  @api expression;
+  /** @type {string} */
+  @api engineId;
+
```

Comment:
extra line here. don't know if it's intentional

Reply:
oops! not intentional. removed it.